aten/src/ATen/native/mps/MPSFallback.cpp

```diff
@@ -0,0 +1,208 @@
+//  Copyright © 2022 Apple Inc.
+
+#include <ATen/ATen.h>
+#include <ATen/native/mps/Copy.h>
+#include <ATen/core/dispatch/Dispatcher.h>
+#include <ATen/core/op_registration/op_registration.h>
+#include <ATen/native/MathBitsFallback.h>
+#include <ATen/core/ivalue.h>
+#include <ATen/core/stack.h>
+#include <ATen/core/boxing/KernelFunction.h>
+#include <ATen/core/dispatch/Dispatcher.h>
+#include <torch/library.h>
+#include <ATen/Functions.h>
+
+namespace at {
+
+struct MPSFallback {
+  MPSFallback() : key(DispatchKey::MPS), op_name("mps") {}
+  std::vector<at::Tensor> to_cpu(const at::TensorList& tensors) {
+      // We can't just call at::to_cpu() on the entire list of Tensors
+      // Because it will break on undefined tensors. Separate out undefined tensors first.
+      std::vector<at::Tensor> cpu_tensors(tensors.size());
+      std::vector<at::Tensor> valid_tensors;
+      std::vector<bool> to_translate(tensors.size());
+      for (const auto i : c10::irange(tensors.size())) {
+          const at::Tensor& tensor = tensors[i];
+          // Explicitly handling undefined tensors here instead of letting `at::_to_cpu` handle it.
```

Comment:
This comment is not really accurate here? This only applies to the mps backend and so we could have the mps copy handle this.
No need to change the logic, just update the comment to be more accurate, and maybe a hinting at the fact that this logic could be moved to the _to_cpu if we wanted to.

Reply:
Done.